modules/ossm-control-plane-remove.adoc

```diff
@@ -39,6 +39,20 @@ You can remove the {SMProductName} control plane by using the CLI.  In this exam
 
 . Log in to the {product-title} CLI.
 
+. Run this command to retrieve the name of the installed `ServiceMeshMemberRoll`:
++
+[source,terminal]
+----
+$ oc get smrr -n istio-system
```

Comment:
typo: smmr

Reply:
By the way, just noticed that we don't need this step as the recource name is always `default`. So we can remove this step and just document the next one, with a hardcoded resource name, e.g.

```
oc delete smmr default -n istio-system
```

Reply:
@jwendell thanks for chatting. I have updated the PR and squashed the job.